layout/_partials/footer.swig

```diff
@@ -35,8 +35,12 @@
 </div>
 
 {% if theme.footer.powered %}
+  {% set hexo_link_tag = '<a class="theme-link" target="_blank" href="https://hexo.io">Hexo</a>' %}
+  {% if theme.seo %}
+    {% set hexo_link_tag = '<a class="theme-link" target="_blank" href="https://hexo.io" rel="external nofollow">Hexo</a>' %}
+  {% endif %}
   <div class="powered-by">{#
-  #}{{ __('footer.powered', '<a class="theme-link" target="_blank" href="https://hexo.io">Hexo</a>') }}{#
```

Comment:
Can u here do like above, without `hexo_link_tag` variable?

Reply:
I have no idea about whether the `{% if %}` tags works within the `__()` as a string.

It is not autocompleted by Atom when I type `if` in the string.

Reply:
Ok, i'll see it later, don't merge.